Dockerfile

```diff
@@ -12,6 +12,9 @@ USER root
 RUN mkdir -p /usr/share/jenkins/ && ln -s /usr/share/cloudbees-jenkins-distribution/cloudbees-jenkins-distribution.war /usr/share/jenkins/jenkins.war
 USER cloudbees-jenkins-distribution
 
+# Startup all plugins included into the CloudBees Jenkins Distribution bundle
+ENV JAVA_OPTS "-Dcom.cloudbees.jenkins.cjp.installmanager.CJPPluginManager.allRequired=${INCLUDE_ALL_BUNDLED_PLUGINS}"
```

Comment:
what if `INCLUDE_ALL_BUNDLED_PLUGINS` has not been set?

Reply:
Fixing it now